core/jazz_asset-event-handler/test/test.js

```diff
@@ -24,46 +24,572 @@ const AWS = require('aws-sdk-mock');
+        return obj.callback(null, responseObject, responseObject.body);
+      });
+      index.processUpdateAsset(record, eventPayload, configData, authToken).catch(err => {
+        sinon.assert.calledOnce(reqStub);
+        expect(err.error).to.eq('Error in updating assets. {"statusCode":0,"body":{"data":{}}}');
+        reqStub.restore();
+      });
+
+    });
+
+    it('process update asset error',() => {
+      responseObject = {
+        statusCode:0,
+        body:{
+          data:{
+          },
+        }
+      };
+      record = {
+        id:121212
+      };
+      eventPayload.EVENT_STATUS.S = 'active';
+      index.processUpdateAsset(record, eventPayload, configData, authToken).catch(err => {
+        sinon.assert.calledOnce(reqStub);
+        expect(err.details).to.eq(eventPayload.EVENT_STATUS.S);
+        reqStub.restore();
+      });
+
+    });
+  })
+
+  describe("processCreateAsset", function (){
+    let responseObject,record;
+    beforeEach(() => {
+		});
+
+		afterEach(() => {
+      reqStub.restore();
+    });
+
+    it('process create asset success',() => {
+      responseObject = {
+        statusCode:200,
+        body:{
+          data:{
+            key:"value"
+          }
+        }
+      };
+      record = {
+        id:121212
+      };
+
+      reqStub = sinon.stub(request, "Request").callsFake((obj) => {
+        return obj.callback(null, responseObject, responseObject.body);
+      });
+      index.processCreateAsset(eventPayload, configData, authToken).then(result => {
+        sinon.assert.calledOnce(reqStub);
+        expect(result).to.eq(responseObject.body);
+        reqStub.restore();
+      });
+    });
+
+    it('process create asset error',() => {
+      responseObject = {
+        statusCode:0,
+        body:{
+          data:{
+          },
+        }
+      };
+      record = {
+        id:121212
+      };
+      reqStub = sinon.stub(request, "Request").callsFake((obj) => {
+        return obj.callback(null, responseObject, responseObject.body);
+      });
+      index.processCreateAsset(eventPayload, configData, authToken).catch(err => {
+        sinon.assert.calledOnce(reqStub);
+        expect(err.error).to.eq('Error in creating assets. {"statusCode":0,"body":{"data":{}}}');
+        reqStub.restore();
+      });
+
+    });
+
+    it('process create asset error',() => {
+      responseObject = {
+        statusCode:0,
+        body:{
+          data:{
+          },
+        }
+      };
+      record = {
+        id:121212
+      };
+      eventPayload.EVENT_STATUS.S = 'active';
+      index.processCreateAsset(eventPayload, configData, authToken).catch(err => {
+        sinon.assert.calledOnce(reqStub);
+        expect(err.details).to.eq(eventPayload.EVENT_STATUS.S);
+        reqStub.restore();
+      });
+    });
+  })
+
+  describe("processItem", function (){
+    var checkIfAssetExistsStub,processUpdateAssetStub,processCreateAssetStub;
+    beforeEach(() => {
+		});
+
+		afterEach(() => {
```

Comment:
here too. please update ian all the places

Reply:
fixed this